src/plugins/inc.rs

```diff
@@ -92,21 +124,21 @@ impl Plugin for Inc {
 
         Ok(Signature {
             name: "inc".to_string(),
-            positional: vec![PositionalType::optional_any("Field")],
+            positional: vec![],
```

Comment:
Curious - why remove the optional positional type?

Reply:
Temporal workaround. You get a parse error when trying #258 and the Signature is configured like that. If we pass in an empty vec it works.